06_Linear-model-sel-reg.Rmd

```diff
@@ -1,211 +1,252 @@
-# Linear Model Selection and Regularization
+---
+editor_options: 
+  markdown: 
+    wrap: 72
+---
```

Comment:
We don't need/want individual-chapter options. Was this needed for something in particular? If it's fine without it, it would be preferable to remove it; we'll be moving everything to quarto soon-ish, so the less things are different the easier that will be.